sdk/servicebus/service-bus/src/util/atomXmlHelper.ts

```diff
@@ -365,7 +375,7 @@ export function buildError(errorBody: any, response: HttpOperationResponse): Res
   }
 
   const error: RestError = new RestError(
-    `ATOM Service Error: ${errorMessage}`,
+    errorMessage,
```

Comment:
The errorMessage currently has code as well. Why do we need to repeat the code in the message when there is a specific place for `code` in the `RestError`?

Reply:
Updated where applicable